pkgs/stdenv/darwin/default.nix

```diff
@@ -280,12 +282,14 @@ in rec {
         coreutils findutils diffutils patchutils;
 
       llvmPackages_5 = super.llvmPackages_5 // (let
-        tools = super.llvmPackages_5.tools.extend (_: _: {
-          llvm = llvmPackages_5.llvm.override { inherit libcxxabi; };
-          clang-unwrapped = llvmPackages_5.clang-unwrapped.override { llvm = self.llvmPackages_5.llvm; };
+        tools = super.llvmPackages_5.tools.extend (llvmSelf: _: {
+          inherit (llvmPackages_5) llvm;
+          # The .override that was here before had the side affect of removing
+          # the hacked-in "man" output.
+          clang-unwrapped = builtins.removeAttrs llvmPackages_5.clang-unwrapped [ "man" ];
```

Comment:
I think the manpages are in a separate drv now, so we can probably get rid of the extra output.

Reply:
For some reason this was causing much more building. Let's investigate this after.

Reply:
@Ericson2314 Looks like this breaks the nixpkgs tarball: https://hydra.nixos.org/build/76089043

Reply:
Fixed on master with #42174 I'm pretty sure.

Reply:
Thanks!